src/components/idleUsers/card/card.module.scss

```diff
@@ -25,9 +25,13 @@ $theme-pink: #e30464;
   }
 
   .name {
-    margin-top: 3%;
-    margin-bottom: 0;
+    display: flex;
     font-size: 1rem;
     font-weight: 700;
+    width: 10rem;
```

Comment:
Instead of setting the width in rem units can we not let the width be the default width (100% of the container) and use padding to reduce the horizontal area? 

This way if a user changes their default scaling from 16px to something bigger, it would not overflow from the card.

Reply:
I tried doing this but it changed the width of the card then

Reply:
Yes, it does, if set 100% the responsibility of handling width should go to the parent. 
Example: [https://zx588j.csb.app/](url)

Reply:
Is it okay now?